apps/ledger-live-mobile/src/datadog.ts

```diff
@@ -82,13 +82,18 @@ const isExcludedErrorDescription = (errorDescription: string): boolean => {
  * @param event The RUMErrorEvent object.
  * @returns The modified RUMErrorEvent object, or null if the event should be dropped.
  */
-export const customErrorEventMapper: ErrorEventMapper = event => {
-  if (isExcludedErrorName(event.stacktrace) || isExcludedErrorDescription(event.message)) {
-    return null; // Return null to drop the event
-  }
+export const customErrorEventMapper: (disableErrorTracking: boolean) => ErrorEventMapper =
```

Comment:
```suggestion
export const customErrorEventMapper = (
  disableErrorTracking: boolean
): ErrorEventMapper => event => {
  const shouldDropEvent =
    disableErrorTracking ||
    isExcludedErrorName(event.stacktrace) ||
    isExcludedErrorDescription(event.message);

  return shouldDropEvent ? null : event;
};
```

I would have wrote something like this, It's more readable I think ? WDYT?

Reply:
maybe if the use of the `shouldDropEvent` will be in the business logic of the entire function, but for this case, it's a free variable reference in the memory for each and everytime the function is called , so why not a oneshot return. Anyways, the error mapper function is well documented to understand that null return will not send the error.